web/cobrands/shropshire/assets.js

```diff
@@ -106,7 +106,6 @@ fixmystreet.assets.add(defaults, {
         }
     },
     asset_group: 'Illuminated signs',
-    feature_code: 'CentralAssetId',
     asset_item: 'bollard',
     actions: {
         asset_found: fixmystreet.assets.named_select_action_found,
```

Comment:
If you're not using feature_code, why are you calling the named_select_action_found/not_found code?

Reply:
I though feature code was for 'construct_selected_asset_message ' and 'asset_item' was for  'named_select_action_found/not_found code'

Reply:
named_select_action_found is to, when an asset is selected, change the message to display the ID of the asset being selected. It calls construct_selected_asset_message, as you can see. asset_item is also used by it (and by other things). If you're not displaying asset IDs when assets are selected, which you don't appear to be, you don't need to call named_select_action_found or similar.